From 4c897cdfa1ade8d1fc61a9c812f48a63a30b3e85 Mon Sep 17 00:00:00 2001
Message-ID: <4c897cdfa1ade8d1fc61a9c812f48a63a30b3e85.1707428203.git.mjg@fedoraproject.org>
From: Michael J Gruber <mjg@fedoraproject.org>
Date: Fri, 2 Feb 2024 22:20:42 +0100
Subject: [PATCH] fix type error with GCC 14

---
 src_classic/fitz_old.i | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src_classic/fitz_old.i b/src_classic/fitz_old.i
index 8c67d2a3..dbd80358 100644
--- a/src_classic/fitz_old.i
+++ b/src_classic/fitz_old.i
@@ -156,7 +156,7 @@ static void show(const char* prefix, PyObject* obj);
 #if FZ_VERSION_MAJOR == 1 && FZ_VERSION_MINOR == 23 && FZ_VERSION_PATCH < 8
 pdf_obj *pdf_lookup_page_loc(fz_context *ctx, pdf_document *doc, int needle, pdf_obj **parentp, int *indexp);
 fz_pixmap *fz_scale_pixmap(fz_context *ctx, fz_pixmap *src, float x, float y, float w, float h, const fz_irect *clip);
-int fz_pixmap_size(fz_context *ctx, fz_pixmap *src);
+size_t fz_pixmap_size(fz_context *ctx, fz_pixmap *src);
 void fz_subsample_pixmap(fz_context *ctx, fz_pixmap *tile, int factor);
 void fz_copy_pixmap_rect(fz_context *ctx, fz_pixmap *dest, fz_pixmap *src, fz_irect b, const fz_default_colorspaces *default_cs);
 void fz_write_pixmap_as_jpeg(fz_context *ctx, fz_output *out, fz_pixmap *pix, int jpg_quality);
-- 
2.43.0.710.g31023adb55

