From 010b0aef5ba0218395dac2b082952737f77ce4cb Mon Sep 17 00:00:00 2001
Message-ID: <010b0aef5ba0218395dac2b082952737f77ce4cb.1784469677.git.mjg@fedoraproject.org>
From: Michael J Gruber <mjg@fedoraproject.org>
Date: Fri, 26 Jun 2026 22:19:57 +0200
Subject: [PATCH] do not test markdown with mupdf < 1.28.0

---
 tests/test_markdown_support.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tests/test_markdown_support.py b/tests/test_markdown_support.py
index 0f7435ee..324e4794 100644
--- a/tests/test_markdown_support.py
+++ b/tests/test_markdown_support.py
@@ -74,6 +74,9 @@ def test_markdown_style():
             assert "Roman" not in spans[0]["font"]
 
 def test_markdown_save():
+    if pymupdf.mupdf_version_tuple < (1, 28, 0):
+        print("no testing on MuPDF < 1.28.0")
+        return
     md = textwrap.dedent('''
             # title
             
-- 
2.55.0.430.g36796e6be8

