From 735e1b2ebc83223287224ab3df08e2e5acb1d05c Mon Sep 17 00:00:00 2001
Message-ID: <735e1b2ebc83223287224ab3df08e2e5acb1d05c.1772136348.git.mjg@fedoraproject.org>
From: Michael J Gruber <mjg@fedoraproject.org>
Date: Tue, 15 Jul 2025 17:21:56 +0200
Subject: [PATCH] adjust tests to tesseract 5.5.1

---
 tests/test_tesseract.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/tests/test_tesseract.py b/tests/test_tesseract.py
index 11c38352..f94557b0 100644
--- a/tests/test_tesseract.py
+++ b/tests/test_tesseract.py
@@ -69,6 +69,8 @@ def test_3842b():
             print(f'test_3842b(): received exception: {e}')
             if 'No tessdata specified and Tesseract is not installed' in str(e):
                 pass
+            elif 'No tessdata specified and no match for:' in str(e):
+                pass
             else:
                 assert 'Tesseract language initialisation failed' in str(e)
 
@@ -87,6 +89,8 @@ def test_3842():
             print(f'test_3842(): received exception: {e}', flush=1)
             if 'No tessdata specified and Tesseract is not installed' in str(e):
                 pass
+            elif 'No tessdata specified and no match for:' in str(e):
+                pass
             elif 'Tesseract language initialisation failed' in str(e):
                 pass
             else:
-- 
2.53.0.546.g660574eaac

