From ea2c10396ce47ce8811713a60bd41b0ee3f0751d Mon Sep 17 00:00:00 2001
Message-ID: <ea2c10396ce47ce8811713a60bd41b0ee3f0751d.1724182862.git.mjg@fedoraproject.org>
From: Michael J Gruber <mjg@fedoraproject.org>
Date: Tue, 20 Aug 2024 16:50:18 +0200
Subject: [PATCH 1/2] use get_tessdata() in test

---
 tests/test_tesseract.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/test_tesseract.py b/tests/test_tesseract.py
index 92212b98..6aa0537e 100644
--- a/tests/test_tesseract.py
+++ b/tests/test_tesseract.py
@@ -35,9 +35,9 @@ def test_tesseract():
         # classic.
         e_expected = 'OCR initialisation failed'
         e_expected_type = None
-    tessdata_prefix = os.environ.get('TESSDATA_PREFIX')
+    tessdata_prefix = pymupdf.get_tessdata()
     if tessdata_prefix:
-        tp = page.get_textpage_ocr(full=True)
+        tp = page.get_textpage_ocr(full=True, tessdata=tessdata_prefix)
         print(f'test_tesseract(): page.get_textpage_ocr() succeeded')
     else:
         try:
-- 
2.46.0.444.g7c79821ace

